site/_data/docs/devtools/toc.yml

```diff
@@ -1,8 +1,10 @@
 - url: /docs/devtools/overview/
 - url: /docs/devtools/open/
-- url: /docs/devtools/updates/
```

Comment:
Do you want to add an entry to redirects.yaml to go from this old URL to the tag page?